src/Common/hooks/useConfig.ts

```diff
@@ -75,6 +75,10 @@ export interface IConfig {
    * Env to enable ABDM features
    */
   enable_abdm: boolean;
+  /**
+   * Env to enable scribe features
+   */
+  enable_scribe: boolean;
```

Comment:
The config change would also have to be updated here: https://github.com/coronasafe/care_deploy_configs